app/src/test/java/com/x8bit/bitwarden/ui/tools/feature/generator/GeneratorViewModelTest.kt

```diff
@@ -986,7 +986,7 @@ class GeneratorViewModelTest : BaseViewModelTest() {
                     includeNumber = false,
                     includeNumberEnabled = true,
 
-                ),
+                    ),
```

Comment:
This formatting was correct before. I think it is the extra newline above that is causing the issue